econml/dml.py

```diff
@@ -333,30 +394,46 @@ class DMLCateEstimator(_DMLCateEstimatorBase):
         If None, the random number generator is the RandomState instance used
         by `np.random`.
 
-    inference: string, inference method, or None
+    inference: string, `Inference` instance, or None
         Method for performing inference.  This estimator supports 'bootstrap'
-        (or an instance of `BootstrapOptions`)
+        (or an instance of `BootstrapInference`) and 'statsmodels' (or an instance of 'StatsModelsInference`)
     """
 
     def __init__(self,
-                 model_y, model_t, model_final=LinearRegression(fit_intercept=False),
+                 model_y=LassoCV(), model_t=LassoCV(),
                  featurizer=PolynomialFeatures(degree=1, include_bias=True),
+                 linear_first_stages=True,
                  discrete_treatment=False,
                  n_splits=2,
-                 random_state=None,
-                 inference=None):
+                 random_state=None):
         super().__init__(model_y=model_y,
                          model_t=model_t,
-                         model_final=model_final,
+                         model_final=StatsModelsWrapper(),
                          featurizer=featurizer,
-                         sparseLinear=False,
+                         linear_first_stages=linear_first_stages,
                          discrete_treatment=discrete_treatment,
                          n_splits=n_splits,
-                         random_state=random_state,
-                         inference=inference)
+                         random_state=random_state)
+
+        self.statsmodelswrapper = self._model_final._model
+
+    def _get_inference_options(self):
+        # add statsmodels to parent's options
+        options = super()._get_inference_options()
+        options.update(statsmodels=StatsModelsInference)
+        return options
 
+    @property
+    def statsmodelsproperties(self):
+        return StatsModelsInference.StatsModelsProperties(self.statsmodelswrapper,
+                                                          self._model_final.effect_op)
+
+    @property
+    def coef_(self):
+        return self.statsmodelswrapper.results.params
```

Comment:
Related to this we might want to expose some method that provides some basic feature names if those are available. For instance the PolynomialFeatures class has a `get_feature_names` method, which we could call. More generally we could call the `get_feature_names` of the featurizer of the final model if such an attribute exist and if not, then raise an error saying that the featurizer does not provide feature names. 

If we enable this, then the user could also get more reasonable names to associate with the `coef_` vector and also with the `coef__interval` results, since each entry corresponds to each variable output by the featurizer and not with a variable of the original X numpy array.

Reply:
I've added a `coef_` property (and added a test for it).  I haven't added `get_feature_names` (but I'd be happy to consider it in a separate PR); one complication is that if we have multiple treatments then we cross the featurized `X`s with all of the `T`s, so the naming is not quite so simple).